packages/cli/lib/prompts/selectPublicAppPrompt.js

```diff
@@ -14,7 +14,7 @@ const fetchPublicAppOptions = async (accountId, accountName) => {
   try {
     const publicApps = await fetchPublicAppsForPortal(accountId);
     const filteredPublicApps = publicApps.filter(
-      app => !app.projectId && !app.sourceId
+      app => !app.projectId && !app.sourceId && !app.listingInfo
```

Comment:
This should work for now. I'll update the app object to have a `preventProjectMigrations` in the future which should land later this month. I'll put it on the app object rather than the `listingInfo` just to future proof this such that we could prevent certain apps from being migrated regardless of marketplace-listing status.